samples/java-customer-registry-quickstart/src/main/proto/customer/api/customer_api.proto

```diff
@@ -51,21 +46,22 @@ message ChangeAddressRequest {
   Address new_address = 2;
 }
 
-// tag::service[]
 service CustomerService {
   option (akkaserverless.service) = {
     type: SERVICE_TYPE_ENTITY
     component: "customer.domain.Customer"
   };
 
-  rpc Create(Customer) returns (google.protobuf.Empty) {}
+  option (akkaserverless.codegen) = {
+    value_entity: {
+      name: "customer.domain.Customer"
+      entity_type: "customers"
+      state: "customer.domain.CustomerState"
+    }
+  };
```

Comment:
Might need some more updates to the docs as well since it now refers to the `customer_domain.proto` annotations.

Reply:
So, if this is being used in the docs, we need to revert it. The goal of this PR was not to change anything that could impact the docs. I wanted to do it in another PR. 

My goal was to have some samples compiled and run with the new annotations.